xds/src/main/java/com/linecorp/armeria/xds/XdsConverterUtil.java

```diff
@@ -52,13 +78,23 @@ static List<Endpoint> convertEndpoints(ClusterLoadAssignment clusterLoadAssignme
                             } else {
                                 return Endpoint.of(socketAddress.getAddress(), socketAddress.getPortValue());
                             }
-                        })).collect(Collectors.toList());
+                        })).collect(toImmutableList());
     }
 
-    static List<Endpoint> convertEndpoints(Collection<ClusterLoadAssignment> clusterLoadAssignments) {
-        return clusterLoadAssignments.stream()
-                                     .flatMap(cla -> convertEndpoints(cla).stream())
-                                     .collect(Collectors.toList());
+    private static boolean containsFilterMetadata(Struct filterMetadata, Struct endpointMetadata) {
+        final Map<String, Value> endpointMetadataMap = endpointMetadata.getFieldsMap();
+        for (Entry<String, Value> entry : filterMetadata.getFieldsMap().entrySet()) {
+            final Value filterMetadataValue = entry.getValue();
+            if (filterMetadataValue == Value.getDefaultInstance()) {
+                // If the value is the default, consider they are the same.
```

Comment:
Question) Is there any documentation that explains this behavior?

Reply:
That was my bad. Removed. Thanks!